test/triplex/triplex_test.exs

```diff
@@ -8,6 +8,8 @@ defmodule TriplexTest do
   @migration_version 20_160_711_125_401
   @repos [PGTestRepo, MSTestRepo]
   @tenant "trilegal"
+  # Added to supress warning about vars used in string interpolation
+  @compile :nowarn_unused_vars
```

Comment:
Not sure if we should suppress warnings at all, I mean, what if other warnings come up and we didn't notice because of that. Ideally we would fix them individually instead of suppressing.

Reply:
Yes ... I am not sure what I was thinking ... had a brain fart with pattern matching in the test. I will fix it.